### ISSUE TITLE
Army not collected nor created.

### ISSUE THREAD

**Issue by DutchGeek** [ThScEo/ForgeOfBot#5]
I ran the program on the en server Dinegu.  I had archer camps needing to be collected, and they were not collected.  Also the training of new troops was not completed.  Premium version.

**Comment by DutchGeek** [comment]
Error when starting military units production

This is the message in the Log window.

**Comment by ThScEo** [comment]
Download the latest release ( I made a patch yesterday) to solve the "Error when starting military units production" message.
I'll try to reproduce your issue. 
As long as this problem is not resolved, all paid licenses are frozen (i.e. the days will not be debited)

**Comment by DutchGeek** [comment]
I downloaded it this morning.  I will try to download again and see if there is any change in version/revision

**Comment by ThScEo** [comment]
The new release uploaded now ( https://github.com/ThScEo/ForgeOfBot/releases/tag/v1.192.2_UI ) should fix all theses issues.

**Comment by ThScEo** [comment]
Can you please confirm or deny that this issue is still relevant?

**Comment by DutchGeek** [comment]
This is still an issue.  I tested with spearfighters, and iron aged troops.

**Comment by ThScEo** [comment]
did you donwload the LATEST release? https://github.com/ThScEo/ForgeOfBot/releases/tag/v1.192.3_UI
Be sure your version is 1.192.3

**Comment by DutchGeek** [comment]
Yes this is the version that I had tested with.  I tried with archers, and with spear fighters.

**Comment by ThScEo** [comment]
Did you set a particular delay time in the options?

**Comment by DutchGeek** [comment]
Did you set a particular delay time in the options?
yes - mainly based on the defaults

Do military buildings have all slots unlocked?
Archers yes (except for the diamond slot) and the spear fighter only 2 slots open, 1 troop generated.

How have you checked that units are not created, via connecting manually in the game or via the bot's interface?  
After the bot is completed, I logged into the game via browser, and checked.  I still had a floating icon above the building.  Collected it manually, and it was the 1st in the building.

Did you seen an error message?
This was the surprising part, I did not see any messages other than completed.

Thank you
